### DISCUSSION TITLE
Can change the variables' names as I prefer?

### DISCUSSION THREAD

**Discussion by chenhaijun02** [seleniumbase/SeleniumBase#2110] [Q&A] [answered]
Hello,Michael
       I have a question,Can change the variables' names as I prefer?
![image](https://github.com/seleniumbase/SeleniumBase/assets/69953502/56cf623a-2b5a-4103-af29-67ae8328e3f7)

**Comment by mdmintz** [comment, marked as answer]
Use ``--variables=DICT`` to pass in custom variables.

Eg: ``pytest --variables='{"MY_VAR":"abc", "VAR123":123}'``

```bash
[1] > HOME/SeleniumBase/examples/test_null.py(7)
   .
   6         def test_null(self):
   7  ->         pass
(Pdb+) self.variables
{'MY_VAR': 'abc', 'VAR123': 123}
(Pdb+) self.variables["MY_VAR"]
'abc'
```
